ldk/javascript/src/whisper/react/renderer-config.ts

```diff
@@ -0,0 +1,330 @@
+    updatePayload: Update | null,
+    type: Type,
+    oldProps: Props,
+    newProps: Props,
+    internalInstanceHandle: any,
+    keepChildren: boolean,
+    recyclableInstance: Instance | null,
+  ): Instance {
+    const value: any = {
+      ...instance,
+      ...newProps,
+    };
+    delete value.children;
+    if (!keepChildren && value.components) {
+      value.components = [];
+    }
+    getHandlerByTagType(type).assignTextChildren?.(instance, newProps);
+    return value;
+  },
+  createContainerChildSet(container: Container): ChildSet {
+    return {
+      label: '',
+      components: [],
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onClose: () => {},
+    };
+  },
+  createInstance(
+    type: Type,
+    props: Props,
+    rootContainer: Container,
+    hostContext: HostContext,
+    internalHandle: Reconciler.OpaqueHandle,
+  ): Instance {
+    const propsWithoutChildren = { ...props };
+    delete propsWithoutChildren.children;
+    const handler = getHandlerByTagType(type);
+    const instance: Instance = {
+      type: handler.helpsType,
+      ...propsWithoutChildren,
+    };
+    handler.assignTextChildren?.(instance, props);
+    if (internalHandle.key) {
+      instance.key = internalHandle.key;
+    }
+    return instance;
+  },
+  createTextInstance(
+    text: string,
+    rootContainer: Container,
+    hostContext: HostContext,
+    internalHandle: Reconciler.OpaqueHandle,
+  ): TextInstance {
+    return {
+      type: WhisperComponentType.Markdown,
+      body: text,
+    };
+  },
+  detachDeletedInstance: undefined,
+  finalizeContainerChildren(container: Container, newChildren: ChildSet): void {
+    // No finalization needs to take place, these objects are ready as-is when generated.
+  },
+  finalizeInitialChildren(
+    instance: Instance,
+    type: Type,
+    props: Props,
+    rootContainer: Container,
+    hostContext: HostContext,
+  ): boolean {
+    // No finalization needs to take place here either;
+    return false;
+  },
+  getChildHostContext(
+    parentHostContext: HostContext,
+    type: Type,
+    rootContainer: Container,
+  ): HostContext {
+    return null;
+  },
+  getCurrentEventPriority: undefined,
+  getInstanceFromNode(node: any): any {
+    throw new Error('Not Implemented');
+  },
+  getInstanceFromScope: undefined,
+  getOffscreenContainerProps(mode: any, children: any): Props {
+    throw new Error('Not Implemented');
+  },
+  getOffscreenContainerType(): string {
+    return '';
+  },
+  getPublicInstance(instance: Instance | TextInstance): Instance {
+    return instance;
+  },
+  getRootHostContext(rootContainer: Container): HostContext | null {
+    return null;
+  },
+  isOpaqueHydratingObject(value: any): boolean {
+    return false;
+  },
+  isPrimaryRenderer: true,
+  makeClientId: undefined,
+  makeClientIdInDEV: undefined,
+  makeOpaqueHydratingObject: undefined,
+  noTimeout: undefined,
+  now(): number {
+    return 0;
```

Comment:
Their documentation states - "You can proxy this to performance.now() or its equivalent in your environment.". Does returning 0 have any side effects?